app/models/cart.rb

```diff
@@ -0,0 +1,3 @@
+class Cart < ApplicationRecord
+  has_many :line_items, dependent: :delete_all
```

Comment:
The book mentions using `dependent: :destroy`,  Is there a specific reason you're using `delete_all`?

Reply:
ahh was fucking with it when testing the `.delete` thing. Good catch!